net/http/src/THttpCallArg.cxx

```diff
@@ -193,6 +193,9 @@ THttpWSEngine *THttpCallArg::TakeWSEngine()
 
 ////////////////////////////////////////////////////////////////////////////////
 /// set binary data, which will be returned as reply body
+/// Memory should be allocated with std::malloc().
```

Comment:
This is terrible in many ways.
- we are C++, we don't do malloc.
- ownership should be explicit through unique_ptr.
- instead of passing `void*` and `length` independently consider passing a vector<char>

Reply:
It is "historical" code and need to be changed in many other places.
Probably, I need to create other PR, which address exactly that issue.

Reply:
Axel, may be we simply can use std::string - it also can store arbitrary binary data of given length. In such case I have generic storage for both kinds of data - text and binary. And only content type can help to identify, how such data should be used.
Are you agree?